docs/user/installconfig.schema.json

```diff
@@ -0,0 +1,417 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://raw.githubusercontent.com/openshift/installer/master/docs/user/installconfig.schema.json",
+  "title": "OpenShift install-config.yaml",
+  "description": "Installer configuration for an OpenShift cluster",
+  "type": "object",
+  "properties": {
+    "apiVersion": {
+      "description": "The API version for the install-config.yaml content. The current version is v1. The installer may also support older API versions.",
+      "type": "string",
+      "exclusiveMinimum": 0
+    },
+    "additionalTrustBundle": {
+      "description": "a PEM-encoded X.509 certificate bundle that will be added to the nodes' trusted certificate store. This trust bundle may also be used when a proxy has been configured",
+      "type": "string"
+    },
+    "additionalTrustBundlePolicy": {
+      "description": "determines when to add the AdditionalTrustBundle to the nodes'' trusted certificate store. \"Proxyonly\" is the default. The field can be set to following specified values. \"Proxyonly\": adds the AdditionalTrustBundle to nodes when http/https proxy is configured. \"Always\": always adds AdditionalTrustBundle.",
+      "type": "string",
+      "enum": [
+        "Proxyonly",
+        "Always"
+      ],
+      "default": "Proxyonly"
+    },
+    "baseDomain": {
+      "description": "The base domain to which the cluster should belong.",
+      "type": "string",
```

Comment:
This is a hostname (rather than a URI since there's no protocol, port or path elements) right? Can we set an appropriate `format` to help editors?

Reply:
indeed, it would be worth checking whether it's `uri` or `hostname` or any of the other choices: https://www.learnjsonschema.com/2020-12/format-annotation/format/